### ISSUE TITLE
bug: Broken SQS support in v3.0.1

### ISSUE THREAD

**Issue by jjmckain** [localstack/localstack#9756]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Since v3.0.1 attempts to list or create queues are failing.  I have not tried other commands.  These work fine on v3.0.0.
`awslocal sqs list-queues`
![image](https://github.com/localstack/localstack/assets/1494042/b81cdfa4-092f-4597-8304-5bf45cf1f583)

`awslocal sqs create-queue ...`
![image](https://github.com/localstack/localstack/assets/1494042/af3e0702-1a4b-47e2-98e1-ecd5d3c130dd)

Noteworthy: This only seems to affect SQS.  My Localstack/AwsLocal S3 commands work fine on either version.

### How are you starting LocalStack?

With a docker-compose file

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    docker run localstack/localstack

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

    awslocal s3 mb s3://mybucket

### Environment

```markdown
- OS: Windows 11
- LocalStack: v3.0.1
- AwsLocal: aws-cli/2.13.37 Python/3.11.6 Windows/10 exe/ prompt/off
```

### Anything else?

Sharing for awareness.  I've gone back to v3.0.0 and everything (I use) works as expected.
Thanks for the great product!

**Comment by alexrashed** [comment]
Hi @jjmckain!
Thanks for creating the issue! This sounds pretty much like https://github.com/localstack/localstack/issues/8267, which should be fixed in `3.0.1`. I also can't reproduce it on my end.
Could it be that your image is just a bit older and you haven't pulled the `lastest` image in a while?
```
docker pull localstack/localstack
```

**Comment by jjmckain** [comment]
I retried v3.0.1 and could not reproduce the problem either.  Perhaps (somehow) I did have a stale v3.0.1 image cached.
I also tested v3.0.2 and am not seeing the issue there either.
